package.json

```diff
@@ -32,61 +32,31 @@
     "stylelint": "^8.0.0 || ^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0"
   },
   "engines": {
-    "node": ">=8"
+    "node": ">=10"
   },
   "eslintConfig": {
     "extends": [
-      "eslint:recommended",
       "stylelint"
     ],
-    "parserOptions": {
-      "sourceType": "script",
-      "ecmaVersion": 6
-    },
-    "env": {
-      "es6": true,
-      "jest": true,
-      "node": true
-    },
     "plugins": [
       "lodash"
     ],
     "globals": {
       "testRule": true
     },
     "rules": {
-      "eqeqeq": "error",
       "no-use-before-define": [
         "error",
         "nofunc"
       ],
-      "sort-requires/sort-requires": "error",
-      "strict": [
-        "error",
-        "global"
-      ],
-      "arrow-spacing": "error",
-      "no-var": "error",
-      "object-shorthand": "error",
-      "prefer-const": "error",
-      "template-curly-spacing": "error",
       "lodash/callback-binding": "error",
       "lodash/collection-method-value": "error",
       "lodash/collection-return": "error",
       "lodash/no-double-unwrap": "error",
       "lodash/no-extra-args": "error",
       "lodash/no-unbound-this": "error",
       "lodash/unwrap": "error",
-      "lodash/preferred-alias": "error",
-      "node/no-unsupported-features/es-syntax": [
```

Comment:
This is inherited by the stylelint eslint config. We could keep it and adapt it if you prefer that.

Reply:
Sorry, what do you mean with "this"?

Reply:
This rule.

Reply:
maybe we should keep it if we want to have a separate minimum node version from stylelint. It should not be a big problem if we inherit it, but we could run into some compatibility issues at some point.

Reply:
I'll have a look later an restore this.